brave/brave-core
# Adds notification for when tips are processed

Fixes: https://github.com/brave/brave-browser/issues/3637
UI: https://github.com/brave/brave-ui/pull/429

The intention is that when AutoContribute is off, that we notify the user when their recurring tips have processed.

<img width="435" alt="Screen Shot 2019-03-14 at 5 33 26 PM" src="https://user-images.githubusercontent.com/8732757/54400543-902ae800-4680-11e9-8544-3cde190bd595.png">

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Start Brave and enable Rewards
2. Claim a grant, turn off auto contribute
3. Make some monthly tips to different publishers
4. Wait a month (or set your reconcile stamp back to something reasonable)
5. Ensure that the notification showing that tips have been processed is in the panel after reconcile is finished.
6. Repeat the steps with auto contribute on, and ensure the notification does not show

## Reviewer Checklist:
